tests/test_validate_cache.py

```diff
@@ -119,9 +120,9 @@ def test_cekit_cannot_add_artifact_without_checksum(tmpdir):
     result = run_cekit_cache(['--work-dir',
                               work_dir,
                               'add',
-                              artifact], 2)
+                              artifact], 1)
 
-    assert "At least one checksum must be provided" in result.output
+    assert "Cannot cache artifact without checksum" in result.output
```

Comment:
changing this test was the only thing that confused me. it seemed to be working before, but after my changes it broke and i'm not sure why. fixing it was just a matter of changing the expected result code and the error text.

Reply:
Added a comment above that points where the issue is.